packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -271,6 +319,11 @@ export class OpenId4VciHolderService {
     let accessTokenResponse: OpenIDResponse<AccessTokenResponse>
 
     const accessTokenClient = new AccessTokenClient()
+
+    const createDPoPOptions = await this.getCreateDPoPOptions(agentContext, metadata)
```

Comment:
In credo we usually remove the capitalization if it's being used aa a word.

So createDpopOptions. Then you always know when a word starts and stops.